src/types/currency.ts

```diff
@@ -36,10 +36,14 @@ export type CollateralIdLiteral =
 const CollateralCurrency = [Polkadot, Kusama, Interlay, Kintsugi] as const;
 type CollateralCurrency = typeof CollateralCurrency[number];
 
-export type ForeignAsset = Currency & {foreignAsset: { id: number; coingeckoId: string }};
-export type LendToken = Currency & {lendToken: { id: number }}; 
+export type ForeignAsset = Currency & { foreignAsset: { id: number; coingeckoId: string } };
+export type LendToken = Currency & { lendToken: { id: number } };
+export type StandardLPToken = Currency & { lpToken: { token0: CurrencyExt; token1: CurrencyExt } };
```

Comment:
Up for discussion: I would prefer `<something optional>LpToken` with lower case `p` rather than capitalized `P`. Even though it stands for `LiquidityPool` (I think?) but makes it just more readable in my opinion.

Reply:
This is good suggestion, actually the runtime uses lowercase p as well, so I changed it +1